it/tutorials/scala-for-java-programmers.md

```diff
@@ -0,0 +1,716 @@
+di codice precedentemente riportato. Un’altra differenza è evidente
+nell’uso del carattere underscore (`_`) al posto dell’asterisco (`*`) per
+importare tutti i nomi di un package o di una classe. Questo perché
+l’asterisco è un identificatore valido (e.g. nome di un metodo), come
+vedremo più avanti.
+
+Inoltre, l’istruzione import sulla terza riga importa tutti i membri
+della classe `DateFormat`. Questo rende disponibili il metodo statico
+`getDateInstance` ed il campo statico `LONG`.
+
+All’interno del metodo `main` creiamo un’istanza della classe `Date` di
+Java che di default contiene la data corrente. Successivamente, definiamo il
+formato della data usando il metodo statico `getDateInstance` importato
+precedentemente. Infine, stampiamo la data corrente, formattata secondo la
+localizzazione scelta, con l’istanza `DateFormat`; quest’ultima linea mostra
+un’importante proprietà di Scala.I metodi che prendono un argomento possono
+essere usati con una sintassi non fissa. Questa forma dell’espressione
+
+    df format now
+
+è solo un altro modo meno esteso di scriverla come
+
+    df.format(now)
+
+Apparentemente sembra un piccolo dettaglio sintattico ma, presenta delle
+importanti conseguenze. Una di queste sarà esplorata nella prossima
+sezione.
+
+A questo punto, riguardo l’integrazione con Java, abbiamo notato che è
+altresì possibile ereditare dalle classi Java ed implementare le interfacce
+direttamente in Scala.
+
+
+## Tutto è un Oggetto
+
+Scala è un linguaggio orientato agli oggetti (_object-oriented_) puro nel
+senso che *ogni cosa* è un oggetto, inclusi i numeri e le funzioni. In questo
+differisce da Java che invece distingue tra tipi primitivi (come `boolean`
+  e `int` ) e tipi referenziati. Inoltre, Java non permette la manipolazione
+  di funzioni come fossero valori.
+
+### I numeri sono oggetti
+
+Poichè i numeri sono oggetti, hanno dei metodi. Di fatti
+un’espressione aritmetica come la seguente:
+
+    1 + 2 * 3 / x
+
+consiste esclusivamente di chiamate a metodi e risulta equivalente alla
+seguente espressione, come visto nella sezione precedente:
+
+    (1).+(((2).*(3))./(x))
+
+Questo significa anche che `+`, `*`, etc. sono identificatori validi in
+in Scala.
+
+Le parentesi intorno ai numeri nella seconda versione sono necessarie
+perché l’analizzatore lessicale di Scala usa le regole di match più
+lunghe per i token quindi, dovrebbe dividere la seguente espressione:
+
+    1.+(2)
+
+nei token `1.`, `+`, and `2`. La ragione per cui si è scelto questo tipo
+di assegnazione di significato è perché `1.` è un match più lungo e valido
+di `1`. Il token `1.` è interpretato come `1.0` rendendolo un `Double` e
+non più un `Int`. Scrivendo l’espressione come:
+
+    (1).+(2)
+
+si evita che `1` sia interpretato come un `Double`.
+
+### Le funzioni sono oggetti
+
+Forse per i programmatori Java è più sorprendente scoprire che in Scala
+anche le funzioni sono oggetti. È pertanto possibile passare le funzioni
```

Comment:
I simply use the original english tutorial, maybe it's time to update the original one :-)